docs/guides/nrfconnect_platform_overview.md

```diff
@@ -0,0 +1,37 @@
+# nRF Connect Platform Overview
+
+The nRF Connect platform is a [CHIP](https://github.com/project-chip/connectedhomeip) platform using Nordic Semiconductor's [nRF Connect SDK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html) and [Zephyr RTOS](https://zephyrproject.org/).
+
+The nRF Connect SDK allows to build a range of cellular IoT (LTE-M and NB-IoT), Bluetooth Low Energy, Thread, Zigbee, and Bluetooth Mesh applications. It containes samples, libraries and full set of drivers for Nordic Semiconductor [nRF9160](https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160), [nRF5340](https://www.nordicsemi.com/Software-and-tools/Development-Kits/nRF5340-PDK) and [nRF52 Series](https://www.nordicsemi.com/Products/Low-power-short-range-wireless) devices.
+
+The nRF Connect SDK is based around Zephyr RTOS, which is scalable real-time operating system, dedicated for the connected, resource constrained devices. Zephyr supports multiple hardware platforms, provides hardware drivers, application protocols, protocol stacks and even more. Besides of the Zephyr, nRF Connect SDK also intergrates other projects like crypto library mbed TLS, MCU Bootloader or Thread stack implementation [OpenThread](https://openthread.io/).
+
+The diagram presented below, shows simplified structure of the CHIP application running on the nRF Connect platform and using BLE and Thread stacks for communication purposes: 
+
+![nrfconnect overview](../../examples/platform/nrfconnect/doc/images/nrfconnect_overview.png)
+
+> **Note**: The diagram doesn't show all projects components, but only the most important for the typical CHIP application, because of the readability reasons.
+
+<hr>
+
+### BLE and Thread stacks
+
+In the nRF Connect platform applications BLE interface is used to perform pairing and Thread network provisioning operations between CHIP device and CHIP controller. After that fully provisioned device is able to communicate with other devices inside of the Thread network.
+
+For the BLE communication purposes application is using BLE stack, in which BLE host part is provided by the Zephyr RTOS and the BLE controller is implemented in the nRF Connect SDK's driver.
```

Comment:
Can we add reference that BLE controller is "SoftDevice Controller" and add a link: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/softdevice_controller/README.html

Reply:
Done